src/google/protobuf/compiler/csharp/csharp_source_generator_base.cc

```diff
@@ -57,7 +59,12 @@ void SourceGeneratorBase::WriteGeneratedCodeAttributes(io::Printer* printer) {
 }
 
 std::string SourceGeneratorBase::class_access_level() {
```

Comment:
@jskeet Do you mean that the `descriptor.proto` can also be generated as `public` and `internal` as opposed to only `internal` now?